src/module/macros/attack.ts

```diff
@@ -384,6 +384,8 @@ export async function checkTargets(
         let target = targetingData.target;
         let actor = target.actor as LancerActor;
         let attack_roll = await new Roll(targetingData.roll).evaluate({ async: true });
+        // @ts-expect-error DSN options aren't typed
+        attack_roll.dice.forEach(d => (d.options.rollOrder = 1));
```

Comment:
Does this need a condition to check whether DSN is enabled, or does it just add an unused `rollOrder` field in that case?

Reply:
It's unused if dsn isn't enabled